autobuild/mac/codeQL/autobuild_mac_2_build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# autobuild_2_build: actual build process
+
+
+####################
+###  PARAMETERS  ###
+####################
+
+source $(dirname $(readlink "${0}"))/../../ensure_THIS_JAMULUS_PROJECT_PATH.sh
+
+###################
+###  PROCEDURE  ###
+###################
+
+cd "${THIS_JAMULUS_PROJECT_PATH}"
+
+
+echo "Building... qmake"
+if [-x /Users/runner/work/jamulus/jamulus/Qt/5.15.2/clang_64/bin/qmake]
```

Comment:
Did you try it without specifying the full path like this?  I found it worked just doing `qmake`, after running the install in the CodeQL workflow.  If it does, it would make it less painful moving from 5.15.2 to 5.9 for the MacOS build.